README.md

```diff
@@ -73,32 +98,32 @@ as `error` level logs.
 
 ##### Methods
 
-#### `logger.report(level, report[, state])`
+##### `logger.info(log[, state])`
```

Comment:
why are we mixing stuff in this PR?

Reply:
why report got changed to info?

Reply:
Actually, `report` is not changed to `info`, in a previous PR we introduced helper functions to do the same without the need of specifying the level of a log as opposed to `report`

Reply:
since it is a version bump, the branch `v0.2.0-alpha` was renamed to `v1.0.0`, all the changes in `v0.2.0-alpha` were already reviewed by you and @dash1291